test/CMakeLists.txt

```diff
@@ -25,82 +25,44 @@ INCLUDE(ExternalProject)
 SET_DIRECTORY_PROPERTIES(PROPERTIES EP_PREFIX ${CMAKE_BINARY_DIR}/test/googletest)
 
 # Add gtest
-OPTION(GOOGLETEST_SVN "Download GoogleTest from the SVN repositoty" OFF)
-IF(GOOGLETEST_SVN)
-  # Download from SVN repository
-    ExternalProject_Add(
-        googletest
-        SVN_REPOSITORY http://googletest.googlecode.com/svn/trunk/
-        TIMEOUT 10
-        # Force separate output paths for debug and release builds to allow easy
-        # identification of correct lib in subsequent TARGET_LINK_LIBRARIES commands
-        CMAKE_ARGS -DCMAKE_ARCHIVE_OUTPUT_DIRECTORY_DEBUG:PATH=DebugLibs
-                   -DCMAKE_ARCHIVE_OUTPUT_DIRECTORY_RELEASE:PATH=ReleaseLibs
-                   -Dgtest_force_shared_crt=ON
-        # Disable install step
-        INSTALL_COMMAND ""
-        # Wrap download, configure and build steps in a script to log output
-        LOG_DOWNLOAD ON
-        LOG_CONFIGURE ON
-        LOG_BUILD ON)
-ELSE()
-    # find installed in system
-    FIND_PATH(GOOGLETEST_SOURCE_DIR NAMES "src/gtest_main.cc"
```

Comment:
We need the option to use a system folder in case the git repository is down.

Reply:
Sounds good to me. I will use the previous cherry-pick to provide this feature on the feature branch.